.gitignore

```diff
@@ -2,4 +2,7 @@
 /target
 /test
 /out
+/lib
+/config*
+/credentials.json
```

Comment:
Edit your running directory in the IDE running settings so that config and credentials aren't generated in the root of the project, instead of modifying the gitignore

Reply:
done